crates/red_knot_python_semantic/resources/primer/good.txt

```diff
@@ -18,7 +16,6 @@ python-chess
 python-htmlgen
 rich
 scrapy
-strawberry
```

Comment:
I added this package in our fork, as it made use of `@dataclass_transform`. I'll propose to add this upstream.

Reply:
https://github.com/hauntsaninja/mypy_primer/pull/157